pkg/kubelet/kuberuntime/kuberuntime_container_windows.go

```diff
@@ -97,7 +97,7 @@ func (m *kubeGenericRuntimeManager) generateWindowsContainerConfig(container *v1
 	if wc.Resources.CpuCount > 0 {
 		if wc.Resources.CpuMaximum > 0 {
 			wc.Resources.CpuMaximum = 0
-			klog.Warningf("Mutually exclusive options: CPUCount priority > CPUMaximum priority on Windows Server Containers. CPUMaximum should be ignored")
+			klog.InfoS("Mutually exclusive options CPUCount and CPUMaximum present, CPUMaximum value ignored")
```

Comment:
Why has the error message been modified here?

Reply:
I was trying to simplify the message somehow, but I doesn't look good to me today. It's probably best to leave the original message intact. I reverted my changes.